src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions/test/InjectDirectiveTest.cs

```diff
@@ -193,9 +196,7 @@ private DocumentIntermediateNode CreateIRDocument(RazorEngine engine, RazorCodeD
             }
         }
 
-        var irDocument = codeDocument.GetDocumentIntermediateNode();
-        irDocument.DocumentKind = MvcViewDocumentClassifierPass.MvcViewDocumentKind;
-        return irDocument;
+        return codeDocument.GetDocumentIntermediateNode();
```

Comment:
Is this no longer needed?

Reply:
The tests weren't registering the `MvcViewDocumentClassifierPass` so the document type wasn't set. I changed the tests to include that pass so there's no need to manually override it anymore.

Reply:
Thanks for the information!

Sigh... This sort of test infrastructure is odd to me. Why not just run the tests on the real thing instead of hacking and setting random properties to make it seem as if it's the real thing? I saw something very similar in tests when removing the `CodeRenderingContext.Items` property.